repos/spack_repo/builtin/packages/nalu_wind/package.py

```diff
@@ -74,6 +74,7 @@ class NaluWind(CMakePackage, CudaPackage, ROCmPackage):
     depends_on("mpi")
     depends_on("yaml-cpp@0.6.0:0.7.0")
     depends_on("kynema", when="+kynema")
+    depends_on("trilinos@master", when="+kynema")
```

Comment:
```suggestion
    depends_on("trilinos@master:", when="+kynema")
```
Probably want to make this a range so `@develop` can be used too

Reply:
Actually I'd probably do it from the next possible trilinos release so we don't have to come back and change it

Reply:
How does that work? If the next trilinos release doesn't exist, will spack automatically pick master instead?

Reply:
Yes exactly that. It's a lower bound and doesn't have to exist.